s4e-web/src/app/views/map-view/state/overlay/overlay.model.ts

```diff
@@ -14,6 +14,7 @@ export interface Overlay {
   id: string;
   type: OverlayType;
   caption: string;
+  favourite?: boolean;
```

Comment:
Overlays do not have this field

Reply:
This will cause incompatibility when some of its children will  inherit also by UILayer